src/Model/QuestionManager.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Model;
+
+class QuestionManager extends AbstractManager
+{
+    
+    const TABLE = 'question';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    /**
+     * Selects a random question.
+     * If table is empty or does not exist or if we encounter an error, returns an error message.
+     * @return array
+     */
+    public function selectOneRandom(): array
+    {
+        try {
+            $statement = $this->pdo->query("SELECT * FROM " . self::TABLE . " ORDER BY RAND() LIMIT 1");
+        } catch (\Exception $e) {
+            return ["id" => 0 , "question" => "Je n'ai pas trouvé de question (╥﹏╥) Essaies d'en ajouter une? ^_^"];
```

Comment:
Why not...
I'd had some kind of error code (empty array, null, Exception, etc...) and if needed display an error message in the view instead of having a fake question containing some error message to be displayed